Tasks/Kvaskov/Task2/Task2.py

```diff
@@ -0,0 +1,64 @@
+from datetime import datetime
+
+dt1 = {0:"Ноль", 1:"Один", 2:"Два", 3:"Три", 4:"Четыре", 5:"Пять", 6:"Шесть", 7:"Семь", 8:"Восемь",
+       9:"Девять", 10:"Десять", 11:"Одиннадцать", 12:"Двенадцать", 13:"Тринадцать", 14:"Четырнадцать",
+       15:"Пятнадцать", 16:"Шестнадцать", 17:"Семнадцать", 18:"Восемнадцать", 19:"Девятнадцать", 20:"Двадцать",
+       21:"Двадцать один", 22:"Двадцать два", 23:"Двадцать три", 24:"Двадцать четыре", 25:"Двадцать пять", 26:"Двадцать шесть",
+       27:"Двадцать семь", 28:"Двадцать восемь", 29:"Двадцать девять"}
+dt2 = {59:"Одной", 58:"Двух", 57:"Трех", 56:"Четырех", 55:"Пяти", 54:"Шести", 53:"Семи", 52:"Восьми", 51:"Девяти", 50:"Десяти",
+       49:"Одинадцати", 48:"Двенадцати", 47:"Тринадцати", 46:"Четырнадцати", 45:"Пятнадцати"}
+dt3 = {1:"Одна", 2:"Две", 3:"Три", 4:"Четыре", 5:"Пять", 6:"Шесть", 7:"Семь", 8:"Восемь", 9:"Девять"}
+dt4 = {31:"тридцать одна", 32:"тридцать две", 33:"тридцать три", 34:"тридцать четыре", 35:"тридцать пять", 36:"тридцать шесть",
+       37:"тридцать семь", 38:"тридцать восемь", 39:"тридцать девять", 40:"сорок", 41:"сорок одна", 42:"сорок две",
+       43:"сорок три", 44:"сорок четыре", 45:"сорок пять"}
+h1 = {1:"час", 21:"час", 2:"часа", 3:"часа",4:"часа", 22:"часа", 23:"часа", 24:"часа", 12:"часов", 5:"часов", 6:"часов",
+      7:"часов", 8:"часов", 9:"часов", 10:"часов",11:"часов", 12:"часов", 13:"часов", 14:"часов", 15:"часов", 16:"часов", 17:"часов",
+      18:"часов", 19:"часов", 20:"часов"} # 1,21 час: 2, 3, 4, 22, 23, 24  часа - родительный падеж, единственное число: 5-20 часов - родительный падеж, множественное число
+
+h2 = {0:"ровно", 30:"половина"} # Один час ровно, половина часа
+h3 = {0:"нулевого", 1:"первого", 2:"второго", 3:"третьего", 4:"четвертого", 5:"пятого", 6:"шестого", 7:"седьмого", 8:"восьмого",
+      9:"девятого", 10:"десятого", 11:"одинадцатого", 12:"двенадцатого", 13:"тринадцатого", 14:"четырнадцатого", 15:"пятнадцатого", 16:"шестнадцатого", 17:"семнадцатого",
+      18:"восемнадцатого", 19:"девятнадцатого", 20:"дватцатого", 21:"двадцатьпервого", 22:"двадцатьвторого",23:"двадцатьтретьего",24:"нулевого"}
```

Comment:
a very thorough data structure, but a bit overcomplicated, from the other hand it leads to a simpler solution logic